pkg/eventemitter/eventemitter.go

```diff
@@ -30,6 +31,8 @@ const (
 	ModifiedReason    = "Modified"
 )
 
+const GetErrorMessage = "Failed to get NetworkAddonsConfig in order to emit event. "
```

Comment:
there is an extra space in the Error message. is it needed?

Reply:
I thought so because of the formatting. now that it is with Wrapf I don't think it is needed.